tools/extra/mmlink/main.cpp

```diff
@@ -116,7 +113,7 @@ void print_err(const char *s, fpga_result res)
 
 void mmlink_sig_handler(int sig)
```

Comment:
You can use this function prototype to avoid compiler warning on unused var:

void mmlink_sig_handler(int)

Reply:
thanks, will update